Hackney.Core/Hackney.Core.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
This should be netstandard2.1. And the output type should be removed. (That github page is wrong.)

Reply:
I've made this change